tests/integ/sagemaker/jumpstart/model/test_jumpstart_model.py

```diff
@@ -86,6 +86,31 @@ def test_prepacked_jumpstart_model(setup):
     assert response is not None
 
 
+def test_default_payload_jumpstart_model(setup):
+
+    # DO NOT COMMIT THIS LINE
+    os.environ.update({"AWS_JUMPSTART_CONTENT_BUCKET_OVERRIDE": "jumpstart-cache-alpha-us-west-2"})
```

Comment:
please don't expose this bucket name in publicly available code.

Reply:
This won't get committed, but yes I will remove